src/containers/pages/InterventionPlan/IRS/plan/index.tsx

```diff
@@ -1709,10 +1711,14 @@ class IrsPlan extends React.Component<
   private onSavePlanButtonClick(e: MouseEvent, isFinal: boolean = false) {
     const { newPlan, childlessChildrenIds } = this.state;
     if (newPlan && newPlan.plan_jurisdictions_ids) {
-      const now = new Date();
+      const now = moment(new Date());
+      const start = moment(newPlan.plan_effective_period_start);
+      const end = moment(newPlan.plan_effective_period_end);
       const newPlanDraft: PlanRecord = {
         ...newPlan,
-        plan_date: `${now.getFullYear()}-${now.getMonth()}-${now.getDate()}`,
+        plan_date: now.format('YYYY-MM-DD'),
```

Comment:
Can we use the `DATE_FORMAT` imported from settings please?  You may need to convert to uppercase for it to work here

Reply:
Ooh yes, I will do.